Sources/SnapshotTesting/AssertSnapshot.swift

```diff
@@ -272,10 +272,25 @@ public func verifySnapshot<Value, Format>(
       }
 
       let testName = sanitizePathComponent(testName)
-      let snapshotFileUrl =
-        snapshotDirectoryUrl
-        .appendingPathComponent("\(testName).\(identifier)")
-        .appendingPathExtension(snapshotting.pathExtension ?? "")
+
+         // Check the bundle for the resource first, then the file system
+         // But, if we're recording, don't bother checking the bundle, since we aren't comparing it to anything, and
+         // want the new file to be generated in the source directory, not the bundle.
+         var snapshotFileUrlCandidate: URL?
+        if record != .never {
```

Comment:
Wouldn't we want to check the bundle if the recording is `never` or `missing`? Because we'd need to know if the snapshot existed before deciding whether to record...   So, this comparison seems incorrect - should it be `record != .always`?  Then, if the file isn't found in the bundle, create it in the snapshot directory.

Reply:
Yeah, good call. Will change.